sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/README.md

```diff
@@ -64,6 +64,7 @@ As a first step you'll need to:
    - In the **Supported account types** section, select **Accounts in any organizational directory**.
    - Add `http://localhost:8080` as the `Reply URL` under Redirect URI.
    - Select **Register** to create the application.
+   - After creating the application, on the application **Overview** page, click the **Redirect URIs** to edit, select the **access token** and **ID token**, and finally click **Save**.
```

Comment:
Please change access token -> `Access tokens` and ID token -> `ID tokens` to be consistent with the portal.

And we'd better remove `finally` in the sentence.